NEWS.md

```diff
@@ -19,6 +19,7 @@
 
 * `object_name_linter()` no longer errors when user-supplied `regexes=` have capture groups (#2188, @MichaelChirico).
 * `.lintr` config validation correctly accepts regular expressions which only compile under `perl = TRUE` (#2375, @MichaelChirico). These have always been valid (since `rex::re_matches()`, which powers the lint exclusion logic, also uses this setting), but the new up-front validation in v3.1.1 incorrectly used `perl = FALSE`.
+* `.lintr` configs set by option `lintr.linter_file` or environment variable `R_LINTR_LINTER_FILE` can point to specific package directories (#2512, @MichaelChirico).
```

Comment:
... can point to subdirectories?

NB the change also affects search from HOME, not only from working directories.

Reply:
not 100% sure I captured your comment, PTAL